hipify_cli.py

```diff
@@ -57,23 +56,63 @@ def main():
         help="The file to Store the return dict output after hipification",
         required=False)
 
-    args = parser.parse_args()
-    print(args)
+    parser.add_argument(
+        '--config-json',
+        type=str,
+        default="",
```

Comment:
Should this be defaulting to empty string? I think it shouldn't have a default, right?